Azure/Azure-Sentinel
# CCP Integration in Packaging Tool

Required items, please complete
   
   Change(s):
   - CCP integration in V3, V4 packaging tool
   - some minor fixed

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

This is already tested by creating package using this tool for Ermes and Okta solution. For Ermes customer confirmed that the package is working. For Okta, Oded confirmed its working and is in partner center for private preview.
